### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Connect to a MySQL Server_I want to connect to a MySQL Server by providing server and authentication details so that I can manage it_VoiceOver: Voiceover is not announcing the Headings present in the "Connections" Dialog.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/azuredatastudio-mysql#196]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #GH-MySQLextensionforAzureDataStudio-MAC-Mar23; #MySQL extension for Azure Data Studio; #Screenreader; #VoiceOver; #Benchmark; #WCAG1.3.1; #Info and Relationships; #Benchmark; #FTP; #MAC; #AILimited; #Element:Heading; #A11yWCAG2.1; #Closed;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Application Name: Azure Data Studio
Application Version: 14.2.0
Operating system: macOS Ventura (Version 13.3)
Screen Reader: Voiceover.
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till new connection and press enter
4. Enter a valid server name username and password and press connect
### Repro Steps:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Select server type as MySQL.
5. Turn on voiceover.
6. Press Control + Option + Command + H
7. Now check that Voiceover is not announcing the Headings or not.
### Actual Result:
Voiceover is not announcing the Headings.
### Expected Result:
Headings Should be defined in the "Connection" menu Pop-up.
### User Impact:
Screen Reader Users will not be able to understand the structure of the screen.
### Attachment:

https://user-images.githubusercontent.com/91586743/229099780-d85fef7c-5830-4b01-907d-ce7c4b9ff947.mov

**Comment by Raghu-addepalli** [comment]
closing this bug as it is expected behavior.
